docs/articles/new-expensify/get-paid-back/Distance-Requests.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Distance Requests
+description: How to create a distance request and request reimbursement for mileage
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# Overview
+
+Expensify allows you to request reimbursement for mileage by creating a distance request from a map. You can send a distance request in Expensify's mobile app, desktop app, or web app.
+
+
+# How to create and send a distance request
+
+1. Click the green '+' button and select Request Money.
+2. Select Distance along the top row of the Request Money window.
+3. Enter the Start and Finish addresses then click Next. If there are multiple stops, you can add them before clicking Next.
+4. Select your organization's workspace from the list of recent workspaces.
+5. On the confirmation page, confirm the amount, Date, and Distance, and optionally enter a Description and Category. Click the Request button.
```

Comment:
NAB: Instead of listing these fields exhaustively, maybe we can be a bit more vague to future proof it? We're going to add a separate rate field and probably others like "Tag" in the future. Something like "Confirm the details and add extra information such as a Description or Category".

Reply:
Updated.

Reply:
Thanks